addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2119,10 +2146,9 @@ var SnippetsMenu = Widget.extend({
                 this.customizePanel.removeChild(this.customizePanel.firstChild);
             }
             $(this.customizePanel).append(content);
-
-            if (tab === this.tabs.OPTIONS) {
-                this._addToolbar();
-            }
+        }
+        if (content && tab === this.tabs.OPTIONS || this.options.enableTranslation) {
+            this._addToolbar();
```

Comment:
See the previous comment with the toolbar being added more than necessary. 
If you still need to update the toolbar, then you need to clear the html of the container.

Reply:
:+1: